docs_guidelines/apidocs/tutorials.guidelines.rst

```diff
@@ -0,0 +1,110 @@
+####################
+Tutorials Guidelines
+####################
+
+1. Get the user started
+=======================
+
+1.1 Lower the barrier of entry 
+------------------------------
+
+Lower the barrier of entry as much as possible. Assume the user does not know much about what is
+covered in the tutorial. If we are expecting users to know something, make it explicit and link them
+to the needed explanations (See :ref:`tutorial-guidelines-minimum-explanation`) Avoid jargon as much as possible. Provide a minimum
+necessary explanation when the jargon is introduced. (See :ref:`tutorial-guidelines-minimum-explanation`)
+ 
+1.2 Provide setup / prerequisites
+---------------------------------
```

Comment:
Should there be text under headings 1.2 and 1.3? I think a bit more explanation about what you mean by setup/pre-reqs would be good